### PR TITLE
Create monero-GUI-guide_es.md

### PR DESCRIPTION
**Pull request by febrezo**

### CODE REVIEW

**Comment by erciccione** [comment]
@lh1008 @blacklotus64 Care to leave a review?

Thanks @febrezo :)

File: translations/monero-GUI-guide_es.md [hunk 1]
```diff
@@ -0,0 +1,331 @@
...
+**(2) Clave de vista secreta:** Las claves de vista secretas permiten al titular ver las transacciones entrantes de su monedero, pero no las salientes. A veces esta funcionalidad es útil para permitir que un tercero pueda auditar y comprobar las transacciones entrantes.    
+**(3) Clave de vista pública:** Las claves de vista públicas se usan para la creación de direcciones ocultas. _Obtén más información sobre las claves en [este enlace [EN]](https://getmonero.org/resources/moneropedia/viewkey.html)._    
+**(4) Clave secreta de gastos:** [**NO comparta su clave secreta de gastos con nadie.**] La clave de gasto secreta se usa para firmar transacciones y debe protegerse con un nivel de seguridad equivalente al de su semilla mnemotécnica.    
+**(5) Clave de gasto público:** La red utiliza la clave de gasto pública para verificar la firma de la imagen de clave que se genera cuando realiza una transacción. Esto evita la presencia del doble gasto ya que la red obliga a que una clave de imagen solo pueda gastarse una vez. _Obtén más información sobre las claves de gasto en [este enlace](https://getmonero.org/resources/moneropedia/spendkey.html)._
```
Comment (lh1008) [inline translations/monero-GUI-guide_es.md]: Just to give it some thoughts, not sure about this translation:

"Clave de gasto pública" or "Clave pública de gasto" 

And the same with the link: https://getmonero.org/es/resources/moneropedia/spendkey.html

File: translations/monero-GUI-guide_es.md [hunk 2]
```diff
@@ -0,0 +1,331 @@
...
+**(4) Clave secreta de gastos:** [**NO comparta su clave secreta de gastos con nadie.**] La clave de gasto secreta se usa para firmar transacciones y debe protegerse con un nivel de seguridad equivalente al de su semilla mnemotécnica.    
+**(5) Clave de gasto público:** La red utiliza la clave de gasto pública para verificar la firma de la imagen de clave que se genera cuando realiza una transacción. Esto evita la presencia del doble gasto ya que la red obliga a que una clave de imagen solo pueda gastarse una vez. _Obtén más información sobre las claves de gasto en [este enlace](https://getmonero.org/resources/moneropedia/spendkey.html)._    
+**(6) Exportar billetera gastable:** [**NO compartas tu código QR del monedero con nadie. Esto puede usarse como una semilla mnemotécnicas para recuperar su billetera.**] Esta opción crea un código QR que contiene todas sus claves.    
+**(7) Exportar solo billetera vista:** Esta opción crea un código QR que contiene solo las claves para ver las transacciones que esta billetera envía o recibe, pero no puede crear y generar transacciones.
```
Comment (lh1008) [inline translations/monero-GUI-guide_es.md]: "Exportar solo billetera vista", you are missing _"Exportar solo billetera **de** viste"_. BTW, I suggest changing every "billetera" word to "monedero".

File: translations/monero-GUI-guide_es.md [hunk 3]
```diff
@@ -0,0 +1,331 @@
...
+**(5) Clave de gasto público:** La red utiliza la clave de gasto pública para verificar la firma de la imagen de clave que se genera cuando realiza una transacción. Esto evita la presencia del doble gasto ya que la red obliga a que una clave de imagen solo pueda gastarse una vez. _Obtén más información sobre las claves de gasto en [este enlace](https://getmonero.org/resources/moneropedia/spendkey.html)._    
+**(6) Exportar billetera gastable:** [**NO compartas tu código QR del monedero con nadie. Esto puede usarse como una semilla mnemotécnicas para recuperar su billetera.**] Esta opción crea un código QR que contiene todas sus claves.    
+**(7) Exportar solo billetera vista:** Esta opción crea un código QR que contiene solo las claves para ver las transacciones que esta billetera envía o recibe, pero no puede crear y generar transacciones.    
+
+# 7. Verificación de los binarios
+
+Debes asegurarte de que los ficheros que has descargado se corresponden con los ficheros oficiales.
```
Comment (lh1008) [inline translations/monero-GUI-guide_es.md]: s/se corresponden, to: **"correspondan"**.

The same here, "ficheros", to: **"archivos"**

File: translations/monero-GUI-guide_es.md [hunk 4]
```diff
@@ -0,0 +1,331 @@
...
+**(5) Clave de gasto público:** La red utiliza la clave de gasto pública para verificar la firma de la imagen de clave que se genera cuando realiza una transacción. Esto evita la presencia del doble gasto ya que la red obliga a que una clave de imagen solo pueda gastarse una vez. _Obtén más información sobre las claves de gasto en [este enlace](https://getmonero.org/resources/moneropedia/spendkey.html)._    
+**(6) Exportar billetera gastable:** [**NO compartas tu código QR del monedero con nadie. Esto puede usarse como una semilla mnemotécnicas para recuperar su billetera.**] Esta opción crea un código QR que contiene todas sus claves.    
+**(7) Exportar solo billetera vista:** Esta opción crea un código QR que contiene solo las claves para ver las transacciones que esta billetera envía o recibe, pero no puede crear y generar transacciones.    
+
+# 7. Verificación de los binarios
+
+Debes asegurarte de que los ficheros que has descargado se corresponden con los ficheros oficiales. 
+Puedes utilizar [esta sencilla guía ilustrada](https://getmonero.org/resources/user-guides/verification-windows-beginner.html) para sistemas Windows.
```
Comment (lh1008) [inline translations/monero-GUI-guide_es.md]: Spanish link: https://getmonero.org/es/resources/user-guides/verification-windows-beginner.html

Comment (lh1008) [inline translations/monero-GUI-guide_es.md]: s/"esta sencilla guía ilustrada", to: **"esta guía paso-a-paso"**

File: translations/monero-GUI-guide_es.md [hunk 5]
```diff
@@ -0,0 +1,331 @@
...
++ **[[EN] La transacción está bloqueada como _pendiente_ en la interfaz gráfica](https://monero.stackexchange.com/questions/6649/transaction-stuck-as-pending-in-the-gui)**
++ **[[EN] La interfaz gráfica parece que tiene errores o se congela todo el rato](https://monero.stackexchange.com/questions/6651/my-gui-feels-buggy-freezes-all-the-time)**
++ **[[EN] Mi nombre contiene caracteres especiales (non-ASCII) (e.g. é, ø, â, Ö) y no puedo crear un monedero en la interfaz gráfica](https://monero.stackexchange.com/questions/6823/my-name-contains-a-special-non-ascii-character-e-g-%c3%a9-%c3%b8-%c3%a2-%c3%96-and-i-cant-c)**
++ **[[EN] La interfaz gráfica usa todo mi ancho de banda y no puedo navegar o usar otras aplicaciones que requieran conexión a internet](https://monero.stackexchange.com/questions/6653/the-gui-uses-all-my-bandwidth-and-i-cant-browse-anymore-or-use-another-applicat)**
++ **[[EN] ¿Cómo puedo mover la cadena de bloques (`data.mdb`) a un directorio diferente durante (o después) de la sincronización inicial sin perder los avances?](https://monero.stackexchange.com/questions/7225/how-do-i-move-the-blockchain-data-mdb-to-a-different-directory-during-or-afte)**
```
Comment (lh1008) [inline translations/monero-GUI-guide_es.md]: s/los avances, to: **"el progreso"**

File: translations/monero-GUI-guide_es.md [hunk 6]
```diff
@@ -0,0 +1,331 @@
...
++ **[[EN] ¿Cómo puedo cambiar el idioma de las 25 palabras de la semilla mnemotécnica en la interfaz gráfica y en la terminal?](https://monero.stackexchange.com/questions/7373/how-do-i-change-the-language-of-the-25-word-mnemonic-seed-in-the-gui/)**
++ **[[EN] Mi cadena de bloques está bloqueada, ¿cómo la desatasco?](https://monero.stackexchange.com/questions/4462/my-blockchain-is-stuck-how-do-i-unstuck-it)**
++ **[[EN] Estoy usando un nodo remoto pero, ¿por qué sigue sincronizándose la cadena de bloques en la interfaz gráfica?](https://monero.stackexchange.com/questions/6324/using-remote-node-still-syncs-blockchain)**
++ **Utilizo una pantalla de alta resolución y la interfaz gráfica parece extremadamente pequeña**    
+Este problema será resuelto próximamente, pero se puede solucionar temporalmente en Windows: haz clic con botón derecho en `monero-wallet-gui.exe`, selecciona `Propiedades` --> `Compatibilidad`. En la opción de alta resolución (`high DPI`), cambia el valor de `Aplicación` a `Sistema` o viceversa.
```
Comment (lh1008) [inline translations/monero-GUI-guide_es.md]: Not sure if this was on purpose but the last point is missing translation:

- TO DO: [How to add the blockchain manually](https://github.com/monero-ecosystem/monero-GUI-guide/tree/master/translations)

File: translations/monero-GUI-guide_es.md [hunk 7]
```diff
@@ -0,0 +1,331 @@
...
++ **[[EN] La interfaz gráfica usa todo mi ancho de banda y no puedo navegar o usar otras aplicaciones que requieran conexión a internet](https://monero.stackexchange.com/questions/6653/the-gui-uses-all-my-bandwidth-and-i-cant-browse-anymore-or-use-another-applicat)**
++ **[[EN] ¿Cómo puedo mover la cadena de bloques (`data.mdb`) a un directorio diferente durante (o después) de la sincronización inicial sin perder los avances?](https://monero.stackexchange.com/questions/7225/how-do-i-move-the-blockchain-data-mdb-to-a-different-directory-during-or-afte)**
++ **[[EN] ¿Cómo puedo cambiar el idioma de las 25 palabras de la semilla mnemotécnica en la interfaz gráfica y en la terminal?](https://monero.stackexchange.com/questions/7373/how-do-i-change-the-language-of-the-25-word-mnemonic-seed-in-the-gui/)**
++ **[[EN] Mi cadena de bloques está bloqueada, ¿cómo la desatasco?](https://monero.stackexchange.com/questions/4462/my-blockchain-is-stuck-how-do-i-unstuck-it)**
```
Comment (lh1008) [inline translations/monero-GUI-guide_es.md]: mmm.. "despego"? :stuck_out_tongue_closed_eyes: Not sure on this.

File: translations/monero-GUI-guide_es.md [hunk 8]
```diff
@@ -0,0 +1,331 @@
...
+
+# 6. Ajustes
+
+La pestaña de `Ajustes` facilita herramientas para configurar las diferentes opciones de la aplicación.
+
+![settings](/media/black_settings.png)
+
+**(1) Cerrar billetera:** Cierra el monedero y reinicia el asistente de creación.    
+**(2) Crear cartera solo de vista:** Crea una versión de solo visualización de la cartera actual. Se te solicitará un nombre de archivo y una contraseña para cifrarla.    
+**(3) Volver a escanear el saldo de la billetera:** Haz un nuevo escaneo de las salidas gastadas.
```
Comment (lh1008) [inline translations/monero-GUI-guide_es.md]: s/Volver a escanear el saldo de la billetera, to: **"Reescanear el balance del monerdero"**

File: translations/monero-GUI-guide_es.md [hunk 9]
```diff
@@ -0,0 +1,331 @@
...
+![settings](/media/black_settings.png)
+
+**(1) Cerrar billetera:** Cierra el monedero y reinicia el asistente de creación.    
+**(2) Crear cartera solo de vista:** Crea una versión de solo visualización de la cartera actual. Se te solicitará un nombre de archivo y una contraseña para cifrarla.    
+**(3) Volver a escanear el saldo de la billetera:** Haz un nuevo escaneo de las salidas gastadas.    
+**(4) Cambiar contraseña:** Cambia la contraseña de su billetera.    
+**(5) Nodo local:** Usa un nodo local y descarga la _blockchain_. Puedes usar un nodo de arranque hasta que tu cadena de bloques esté sincronizada.    
+**(6) Nodo remoto:** Usa un nodo remoto sin descargar la cadena de bloques.    
+**(7) Dirección de inicio / nodo remoto:** Introduce el nombre del _host_ o la dirección IP del nodo remoto.    
+**(8) Bootstrap / Remote Node Port:** Introduce el puerto del nodo remoto.
```
Comment (lh1008) [inline translations/monero-GUI-guide_es.md]: s/Remote Node Port, to: **"Puerto de nodo remoto"**

I believe this was missing.

File: translations/monero-GUI-guide_es.md (hunk 6 above)
Comment (febrezo) [reply translations/monero-GUI-guide_es.md]: Yep, it did not appear in the final English text so I omit that lines found in the placeholder. ;)

File: translations/monero-GUI-guide_es.md (hunk 7 above)
Comment (febrezo) [reply translations/monero-GUI-guide_es.md]: Sorry, I don't understand what's the problem here! ;)

Comment (lh1008) [reply translations/monero-GUI-guide_es.md]: No, you can leave it. Never pointed a problem. Just maybe a different word. "Desatasco" sounds a bit strange where I live but I know is a word that anyone can use.

Comment (BlackLotus64) [reply translations/monero-GUI-guide_es.md]: Hi @febrezo, lh1008 and me are two Spanish translators and we tend to review almost everything hehe, some of the reviews are more like suggestions, it is not a problem or something that must be changed yes or yes.

For this case I agree with lh1008, it does not sound bad and it's technically correct (*desatasco*) but at least in my country is not so much used, so what we mean by making some suggestions is to choose better terminology used in almost all the variants of the language (you know how diverse is the language).

I can suggest to use: * Mi cadena de bloques está bloqueada, ¿cómo la desbloqueo?*

To be honest I haven't read the original file, I'll look to it right now but I wish to leave the comment first, I can edit it later :)

Comment (febrezo) [reply translations/monero-GUI-guide_es.md]: Ok, np! I didn't notice that «desatascar» was the problem. I'm updating it!

**Comment by erciccione** [comment]
@lh1008 @BlackLotus64 If this final version is ok for you. Please leave your approval, then i will merge it

**Comment by lh1008** [comment]
Thank you @febrezo and @BlackLotus64. Great job guys!

@erciccione It's OK with me. :)

**Comment by erciccione** [comment]
Ping @febrezo

**Comment by lh1008** [comment]
Hi guys, 

@febrezo are you working on the changes?

**Comment by Rho-n** [comment]
If you want to accept and merge  this pull request. I will put the translation into the new format.

**Comment by lh1008** [comment]
I think is time to move on :).

**Comment by BlackLotus64** [comment]
I can translate the rest of the guide, I'll get it ready asap and @lh1008 can take a look at it

**Comment by erciccione** [comment]
@Rho-n Was planning to do it myself, but sure, that would be great. @BlackLotus64 You could finish the translation once the guide is recovered in the new format. Merging this now.

**Comment by erciccione** [comment]
#65 is merged

**Pull request merged**